packages/shared/src/components/cards/survey/FeedSurveyCard.tsx

```diff
@@ -22,13 +22,12 @@ export function FeedSurveyCard({
 
   return (
     <div
-      className="relative rounded-16 p-px"
+      className="relative rounded-16 p-0.5"
       style={{ backgroundImage: feedSurveyBorder }}
     >
-      <div className="absolute inset-px rounded-16 bg-background-default" />
       <CardComponent
         className={classNames(
-          'relative items-center gap-4 overflow-hidden',
+          'relative items-center gap-4 overflow-hidden !rounded-14 border-0 shadow-none',
```

Comment:
I think we don't have to override the roundness anymore as it is naturally found at the `CardComponent`.
```suggestion
          'relative items-center gap-4 overflow-hidden border-0 shadow-none',
```

Reply:
We still need it, as `CardComponent` has a too rounded radius.
<table>
<tr>
<td>rounded-16
<td>!rounded-14
<tr>
<td><img src=https://github.com/dailydotdev/apps/assets/1681525/43a75766-071a-4905-8f56-b3f353f8ac8d>
<td><img src=https://github.com/dailydotdev/apps/assets/1681525/e2db1cb1-dcbb-4a5f-9598-52a3437b0b7a>

Reply:
`inner-radius = outer-radius - padding`
I.e. 16px (outer radius) - 2px (padding) = 14px